lib/showoff.rb

```diff
@@ -858,7 +858,6 @@ def update_commandline_code(slide)
           lines = out.split("\n")
           if lines.first.strip[0, 3] == '@@@'
             lang = lines.shift.gsub('@@@', '').strip
-            pre.set_attribute('class', 'highlight')
```

Comment:
The example doesn't work for me. Steps:

* Replace

```
@@@javascript
the code
```

 with

````
```javascript
the code
```
````

 in example/one/slidesB.md and add `"markdown": "commonmarker",` to example/showoff.json. Result on http://localhost:9090/#8:

![screenshot from 2017-01-16 12-46-12](https://cloud.githubusercontent.com/assets/650430/21981871/c884cb3e-dbe9-11e6-8cc2-6f21441f97b7.png)